### PR TITLE
Layer3s

### PR DESCRIPTION
**Pull request by inathanb6**
Tested and this works, only issue I've found is "remove comment" doesn't change to "restore comment" when a comment is removed, probably something silly.

### CODE REVIEW

File: src/lemmyEvents/filterHandler.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ export default class FilterHandler {
       console.log("Checking for match!");
       console.log(x);
       const found = checkText(x.words, [
-        postData.post.body || "",
-        postData.post.name,
+        postData.post.body.replace(/\*/g, '') || "",
+        postData.post.name.replace(/\*/g, ''),
```
Comment (Pdzly) [inline src/lemmyEvents/filterHandler.ts:23]: Why is it required that * should be replaced?

Comment (Pdzly) [reply src/lemmyEvents/filterHandler.ts:23]: and then why not on comments?

**Review by Pdzly — CHANGES_REQUESTED** [review]

File: src/lemmyEvents/filterHandler.ts (hunk 1 above)
Comment (inathanb6) [reply src/lemmyEvents/filterHandler.ts]: I removed the * replace on that file

**Comment by inathanb6** [comment]
Committed the changes requested

**Review by Pdzly — APPROVED** [review]

**Pull request merged**